### ISSUE TITLE
Using `useCookie` with `CookieStore` disabled causes an audit failure in lighthouse (bfcache)

### ISSUE THREAD

**Issue by cjpearson** [nuxt/nuxt#28789]
### Environment

Nuxt@3.13.0

### Reproduction

https://stackblitz.com/edit/github-vbpscx-nkrxou

### Describe the bug

This is the same issue as described in #23249

When using `useCookie` within a component or plugin, the Lighthouse bfcache audit fails due with the reason "The page cannot be cached because it has a BroadcastChannel instance with registered listeners."

This is reproducible in 3.11.2 (possibly earlier), and in newer versions if you set `experimental.cookieStore = false`

### Additional context

https://developer.chrome.com/docs/lighthouse/performance/bf-cache

**Comment by danielroe** [comment]
One reason we switched `CookieStore` on by default is in order to work around this issue.

Is there a reason you do not want to use it?

**Comment by cjpearson** [comment]
No, for our app enabling the option should be fine. (And we'd like to bump the Nuxt version soon as well)

Just wanted to report it in case it's unintentional or part of a deeper issue, but it's not blocking us.

**Comment by danielroe** [comment]
Resolving this as CookieStore is enabled by default - this is a known limitation of using BroadcastChannel.
